regtest.sh

```diff
@@ -270,9 +320,12 @@ run-seq() {
 }
 
 run-all() {
+  # Number of iterations of each test.
+  local repetitions=${1:-1}
```

Comment:
run-seq has filters
run-all now has repetitions

both have 2 positional args for filter and repetitions

foo() { pat=${1:-}; num=${2:-1}; echo "PAT $pat NUM $num"; }